MediaGalleryUi/view/adminhtml/web/js/grid/columns/image.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([
+    'jquery',
+    'Magento_Ui/js/grid/columns/image'
+], function ($, image) {
+    'use strict';
+
+    return image.extend({
```

Comment:
Is it possible to override only the `bodyTmpl` and prepare the data for rendering on the backend when populating `media_gallery_asset_grid` table?

Reply:
yes :+1: